data/remnant missions.txt

```diff
@@ -1506,3 +1510,150 @@ mission "Remnant: Broken Jump Drive job"
+	on offer
+		conversation
+			`When the technician arrives in the spaceport he immediately strides over to you. "I checked the records, and my memory was correct. One of our research teams back then discovered that these tentacles could be used as a devastating weapon in close quarters combat."`
+			`	He pulls out an image of a strange looking rifle and continues to chant "the personal weapons lab under Prefect Chilia has a few of the old weapon frames. If you go there, they could turn this tentacle into a fairly potent weapon for you."`
+			choice
+				`	"That sounds good."`
+					accept
+				`	"I would rather not."`
+			`	The technician looks dissapointed. "Oh well, enjoy your trophy" he chants.`
+			decline
+	on complete
+		log "People" "Chilia" `As an military prefect, Chilia is responsible for the defence of Remnant space. He can usually be found on the front lines of combat unless required to work on tactics for the Remnant as a whole.`
+		payment 250000
+		"remnant chilia" ++
+		outfit "Void Rifle"
+		conversation
+			branch chilia
+				"remnant chilia" > 1
+			`You follow the directions provided you and eventually find yourself at a Remnant weapons lab. You enter and are met by security, who ask you to wait. A few minutes later a fit young man emerges from another door "Greetings, captain <first>. I am prefect Chilia. I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly "I say 'nominal' as virtually all our activities are military in some way, but usually fall under the purview of those who better understand them."`
+				goto main
+			label chilia
+			`You follow the directions provided you and eventually find yourself at a Remnant weapons lab. You enter and are met by security, who ask you to wait. A few minutes later a fit young man emerges from another door "Greetings, captain <first>, we meet once again." You immediately recognize the voice as being the man who convinced you to provide the blood sample and get to know the Remnant. "We didn't get a proper introduction last time, but you have probably figured out that I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly "I say 'nominal' as virtually all our activities are military in some way, but usually fall under the purview of those who better understand them."`
+			label main
+			`	He leads you to a room that looks like an armory where a team of techs are putting the finishing touches on a rifle stock. "The request from the tech on your behalf caught my attention, as we haven't had the key resources for making void rifles in centuries. Fortunately, we haven't really needed them since our starships have been brought up to their current capabilities." He gestures, and a tech takes the container with the tentacle from you and starts installing it in a framework.`
+			choice
+				`	"Void rifle?"`
+				`	(expression of boredom)`
```

Comment:
This doesn't follow the usual action choice format. (It should be an actual sentence.)

Reply:
I changed it to "I do not need details."

That being said, would (Make an expression of boredom.) sound better?